.github/workflows/main.yml

```diff
@@ -1,9 +1,14 @@
-name: DVC benchmarks
-on: [push, pull_request]
+name: master build
+on:
+  push:
+    branches:
+      - master
 
 jobs:
-  verify:
-    name: tests and styling check
+  build:
+    # Prevent results update PR from triggering the workflow
+    if: github.event.head_commit.author.name != 'github-actions[bot]'
```

Comment:
verified [here](https://github.com/iterative/dvc-bench/runs/761168233?check_suite_focus=true)

Reply:
Oh, so we don't even need to create a bot user? Github Actions are really a box of treats, aren't they? :slightly_smiling_face:

Reply:
yep, cron jobs are also in

Reply:
If you push using the builtin github token then it seems a job shouldn't be triggered - https://docs.github.com/en/actions/reference/events-that-trigger-workflows#triggering-new-workflows-using-a-personal-access-token